Gauntlet.toml

```diff
@@ -3096,6 +3101,11 @@ document = "broadinstitute/warp:/tests/skylab/smartseq2_multisample/pr_single_en
 message = "test_smartseq2_multisample_PR_SINGLE_END.wdl:58:46: warning[UnusedCall]: unused call `checker_workflow`"
 permalink = "https://github.com/broadinstitute/warp/blob/16affa2fbda7455569e1953ef34d8ded1521890f/tests/skylab/smartseq2_multisample/pr_single_end/test_smartseq2_multisample_PR_SINGLE_END.wdl/#L58"
 
+[[diagnostics]]
+document = "broadinstitute/warp:/tests/skylab/smartseq2_single_nucleus/pr/test_smartseq2_single_nucleus_PR.wdl"
+message = "test_smartseq2_single_nucleus_PR.wdl:28:19: warning: missing required call input `cloud_provider` for workflow `MultiSampleSmartSeq2SingleNucleus`"
+permalink = "https://github.com/broadinstitute/warp/blob/16affa2fbda7455569e1953ef34d8ded1521890f/tests/skylab/smartseq2_single_nucleus/pr/test_smartseq2_single_nucleus_PR.wdl/#L28"
+
```

Comment:
This is also a real diagnostic. The WDL is part of their test framework, so there may be something that specifies this outside the WDL. If I recall correctly, `cromwell` doesn't require the `allowNestedInputs` directive; it just does that by default. So the runner could specify this as part of the JSON.